docs/UserGuide.md

```diff
@@ -140,21 +140,24 @@ Examples:
 
 - `fav 1` — Adds contact at index 1 to your list of favourites
 
-### Adding high importance flag: `impt`
+### Adding high importance flag : `impt`
 
-Adds the contact to your list of people with high importance.
+Adds the contact to your list of people with high importance and a red flag will appear beside the contact's name to indicate that.
```

Comment:
Might want to be consistent by using `contacts` instead of `people`

Reply:
Thanks for the reminder, fixed the required changes in the latest commit! 👍